mysql-test/main/AAA.test

```diff
@@ -0,0 +1,38 @@
+source include/have_innodb.inc;
```

Comment:
erroneously committed file :)

_if this would've been the only issue, I'd removed it myself, but see below about rename_

Reply:
Grrr... sorry, must've mistakenly added during rebase.